tests/test_update.py

```diff
@@ -101,6 +113,9 @@ def test_slot_behaviour(self, update, recwarn, mro_slots):
 
     @pytest.mark.parametrize('paramdict', argvalues=params, ids=ids)
     def test_de_json(self, bot, paramdict):
+        if 'chat_join_request' not in paramdict:
+            return
+
```

Comment:
this intended?

Reply:
No, I'm just sloopy at deleting my debug prints 🙈